client/src/index.css

```diff
@@ -67,11 +67,11 @@ html,
   transition: all 0.3s;
 }
 
-@media only screen and (max-width: 600px) {
+/* @media only screen and (max-width: 600px) {
   .auth-inner {
     width: 100vw;
   }
-}
+} */
```

Comment:
Would you mind deleting the commented code?

Reply:
@laredotornado The comment has been deleted and I added another commit to bring the app to 100vw under 600px per Steve's request.